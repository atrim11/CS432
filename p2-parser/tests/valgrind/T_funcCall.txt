==966694== Memcheck, a memory error detector
==966694== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==966694== Using Valgrind-3.22.0 and LibVEX; rerun with -h for copyright info
==966694== Command: ../decaf inputs/functioncallanddecleration.decaf
==966694== 
Program [line 1]
  FuncDecl name="add" return_type=int parameters={x:int,y:int} [line 1]
    Block [line 2]
      Return [line 3]
        Binaryop op="/" [line 3]
          Location name="x" [line 3]
          Location name="y" [line 3]
==966694== 
==966694== HEAP SUMMARY:
==966694==     in use at exit: 0 bytes in 0 blocks
==966694==   total heap usage: 1,722 allocs, 1,722 frees, 196,944 bytes allocated
==966694== 
==966694== All heap blocks were freed -- no leaks are possible
==966694== 
==966694== For lists of detected and suppressed errors, rerun with: -s
==966694== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
