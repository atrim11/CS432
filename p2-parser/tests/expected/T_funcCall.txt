Program [line 1]
  FuncDecl name="add" return_type=int parameters={x:int,y:int} [line 1]
    Block [line 2]
      Return [line 3]
        Binaryop op="/" [line 3]
          Location name="x" [line 3]
          Location name="y" [line 3]
